twitch4j-ktx/src/main/java/com/github/twitch4j/ktx/main/EventManagerKtx.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.twitch4j.ktx.main
```

Comment:
Just a side note: I think we would also create a `events4j-kotlin` module in https://github.com/PhilippHeuer/events4j and then include this as dep in the new twitch4j-kotlin module when we merge this. But you could create this mr after this is fully reviewed (or alternatively i could also do it as part of the merge).

Reply:
That makes much more sense, yeah!

Reply:
FYI: can we naming like this?
```suggestion
package com.github.twitch4j.kotlinx
```

Reply:
Would it be kotlinx.main/coroutines.main? Or just kotlinx/coroutines?
I wanted them split up in separate packages so every module extensions could be in their own package. Since EventManager is used throughout the project I placed it in a separate package.

Reply:
cc: @PhilippHeuer / @iProdigy